Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM quay.io/centos/centos:stream8
+ARG BASE_IMAGE=quay.io/centos/centos@sha256:5d4193b21043f8b98e9e1e5c39ca8945e7ed816b663febc6092f811f4514062c  # stream9
```

Comment:
this should just be stream9

Reply:
Whoops, pushed the same commit as in #20. Now fixed.